### PR TITLE
Maid 1145

### PR DESCRIPTION
**Pull request by mmoadeli**
[Review on Reviewable](https://reviewable.io/reviews/maidsafe/maidsafe_vault/118)

### CODE REVIEW

**Comment by maidsafe-highfive** [comment]
Thanks for the pull request, and welcome! The MaidSafe team is excited to review your changes, and you should hear from @maqi (or someone else) soon.

If any changes to this PR are deemed necessary, please add them as extra commits. This ensures that the reviewer can see what has changed since they last reviewed the code. The way GitHub handles out-of-date commits, this should also make it reasonably obvious what issues have or haven't been addressed. Large or tricky changes may require several passes of review and changes.

Please see [CONTRIBUTOR.md](../blob/master/CONTRIBUTOR) for more information.

**Comment by benjaminbollen** [comment]
although it should be 'sodiumoxide = "*"'

File: Cargo.toml [hunk 1]
```diff
@@ -15,7 +15,7 @@ docopt = "*"
 rustc-serialize = "*"
 cbor = "*"
 time = "*"
-sodiumoxide = "0.0.4"
+sodiumoxide = "0.0.5"
```
Comment (chandraprakash) [inline Cargo.toml]: Please make it to use sodiumoxide = "*"' as per morning meeting today.

**Comment by chandraprakash** [comment]
Created a separate pr to address this. So closing it.
https://github.com/maidsafe/maidsafe_vault/pull/120

**Pull request closed without merging**